src/Actions/Actions/CaptureAction.cs

```diff
@@ -117,12 +118,13 @@ internal static void PopulateData(ElementDataContext dc, DataContextMode dcMode,
         {
             dc.TreeMode = tm;
             dc.Mode = dcMode;
+            var dataContext = new TreeWalkerDataContext();
```

Comment:
Maybe a TODO to help the future integration?
```suggestion
            var dataContext = new TreeWalkerDataContext(); // TODO: Initialize from parameters
```